default/stringtables/en.txt

```diff
@@ -1147,6 +1147,18 @@ Gas Giants Habitable size
 RULE_HABITABLE_SIZE_DESC
 Value used to adjust for [[encyclopedia POPULATION_TITLE_SHORT_DESC]] changes on a planet of this size.
 
+RULE_THRESHOLD_HUMAN_PLAYER_WIN
+Limit human player winners
+
+RULE_THRESHOLD_HUMAN_PLAYER_WIN_DESC
+Limit count of alive human players for them to be considered winners. Not active for single-player.
```

Comment:
players that can collectively / all / together be considered

Reply:
Fixed.